src/crypto/crypto_cipher.cc

```diff
@@ -993,11 +993,15 @@ void PublicKeyCipher::Cipher(const FunctionCallbackInfo<Value>& args) {
       return ThrowCryptoError(env, ERR_get_error());
     }
 
+#ifndef OPENSSL_IS_BORINGSSL
+    // RSA implicit rejection here is not supported by BoringSSL.
+    // Skip this check when boring is used.
```

Comment:
Won't this make BoringSSL builds vulnerable to [CVE-2023-46809](https://people.redhat.com/~hkario/marvin/)?

Reply:
maybe? This is how electron is using this today in their patch (they're skipping over this). @codebytere

Reply:
I think it'd be best if this was addressed by either adding a comment "Electron is intentionally vulnerable to CVE-2023-46809" or a comment explaining why Electron is not vulnerable despite skipping this check.

I am not going to block this PR because I assume nobody cares much.